LiskHQ/lisk-hub
# Update desktop app icon

### What issue have I solved?

There was a new icon for the desktop app provided by the design team.

### How have I implemented/fixed it?

I used the new icon files.

### How has this been tested?

- Run `npm run pack`.
- Open the `dist/*.dmg` file to see the new icon in the installer and in dock.


### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
